### DISCUSSION TITLE
Docker: Start Azuracast

### DISCUSSION THREAD

**Discussion by echoplex-media** [AzuraCast/AzuraCast#3571] [General]
I understand this is super basic, but I am not a docker user. I have the 1-click install on Digital Ocean, super happy with it. But it occurs to me that I have no idea how to restart Azuracast after a server reboot after updates that require system reboot.

TIA and thanks for making this super great software. Our old Icecast2 server was in bad shape.

**Comment by BusterNeece** [comment]
@echoplex-media It should start on its own, but if it doesn't you can connect via SSH and run, as root or as a user in the `docker` group:

```bash
cd /var/azuracast
docker-compose up -d
```

**Reply by echoplex-media** [reply]
Thanks!
